sql-plugin/src/main/scala/org/apache/spark/sql/rapids/aggregate/aggregateFunctions.scala

```diff
@@ -1660,6 +1664,14 @@ case class GpuNthValue(child: Expression, offset: Expression, ignoreNulls: Boole
     RollingAggregation.nth(offsetVal - 1,
       if (ignoreNulls) NullPolicy.EXCLUDE else NullPolicy.INCLUDE)
         .onColumn(inputs.head._2)
+
+  override def newFixer(): Option[BatchedRunningWindowFixer] = {
+    GpuOverrides.extractLit(offset) match {
```

Comment:
Does this work? Isn't offset going to be a GpuLiteral, which extractLit cannot handle

https://github.com/NVIDIA/spark-rapids/blob/93c03e8a81d1d1b5e1597633396fc18214ab63b6/sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala#L579-L584

Would not be hard to add one that does the same thing for GPU expressions though.

Reply:
Huh. That's a good question.  The tests are working, so I'd better check how.

Reply:
Double-checked this.  `GpuNthValue` has a `Literal(1, IntegerType)`, not a `GpuLiteral`.